docs/2.9.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -645,6 +651,58 @@ the :ref:`transfer command <transfer.execute>`, contracts can be moved over to t
 one by one, such that eventually, all contracts are associated with the new sync domain, allowing
 the old sync domain to be decommissioned and turned off.
 
-The soft upgrade path provides a smooth user experience that does not require a hard migration of the sync domain connection to be 
+The soft upgrade path provides a smooth user experience that does not require a hard migration of the sync domain connection to be
 coordinated across all participants. Instead, participants upgrade individually, whenever they
 are ready, allowing them to reverse the process if needed.
+
+.. _one_step_migration:
+
+One Step Migration
+------------------
+A *one step migration* intends to :ref:`upgrade the binary <upgrade_canton_binary>` and
+:ref:`change the protocol version <change_pv>` in one step. It is a combination of the
+steps documented in :ref:`Hard Synchronization Domain Connection Upgrade <hard_sync_domain_upgrade>`
+and :ref:`Upgrade Canton Binary <upgrade_canton_binary>` from a operational point of
+view. Thus, we assume strong familiarity with aforementioned sections.
```

Comment:
```suggestion
view. Thus, strong understanding of the aforementioned sections is required.
```
Hi, according to our style guide, we're steering away from the "we" or "one" voice and using the "you" voice instead, when necessary. Thank you.